catalogue/webapp/components/ExpandedImage/ExpandedImage.tsx

```diff
@@ -283,7 +283,7 @@ const ExpandedImage: FunctionComponent<Props> = ({
             <NextLink {...expandedImageLink} passHref>
               <ImageLink>
                 <IIIFImage
-                  layout="raw"
+                  layout="fixed"
```

Comment:
I think these changes might be what caused the modal to break

<img width="1060" alt="Screenshot 2023-01-31 at 10 37 29" src="https://user-images.githubusercontent.com/110461050/215737333-9948267c-2b8c-4b9a-84bb-01e443651c7f.png">